fdbclient/BackupContainer.actor.cpp

```diff
@@ -2173,15 +2200,17 @@ ACTOR Future<Void> testBackupContainer(std::string url) {
 					v = nextVersion(v);
 				}
 			}
-			Reference<IBackupFile> range = wait(c->writeRangeFile(snapshots.rbegin()->first, 0, v, 10));
+			Reference<IBackupFile> range = wait(c->writeRangeFile(snapshots.rbegin()->first, 0, v, 16));
```

Comment:
why block size is changed to 16? is it because of `blockSize > 3 * sizeof(uint32_t) + begin.size() + end.size()` assert? 
If not hard/complex, can we calculate a proper blockSize? 
If complex, we can add a comment here to explain.

Reply:
Yes. I'm choosing a size to be big enough to hold two empty keys.